.golangci.yml

```diff
@@ -0,0 +1,17 @@
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters:
+  enable:
+    - exportloopref
+    - gofumpt
+    - misspell
+    - nakedret
+    - revive
+    - prealloc
```

Comment:
copied the celestia-app `.golanci.yml` b/c that seems like a good starting point / uncontroversial